app/javascript/components/ingredients/ingredients-form.vue

```diff
@@ -129,31 +163,48 @@ export default {
         sku: null,
         price: '',
         currency: 'CLP',
-        quantity: '',
-        measure: '',
+        ingredient_measures_attributes: [], /* eslint-disable-line camelcase */
       },
+      showingMeasureModal: false,
+      usedUnits: [],
     };
   },
-
+  methods: {
+    AddMeasure() {
+      const lastItem = this.form.ingredient_measures_attributes[this.form.ingredient_measures_attributes.length - 1];
+      if(lastItem.name && lastItem.quantity) {
+        this.form.ingredient_measures_attributes.push({ name: undefined, quantity: undefined, id: undefined });
+      }
+    },
+    deleteUnit(unit) {
+      this.form.ingredient_measures_attributes = this.form.ingredient_measures_attributes /* eslint-disable-line camelcase */
+        .filter((originalUnit) => originalUnit !== unit);
+    },
+  },
   created() {
     const {
       providerId,
       name,
       sku,
       price,
       currency,
-      quantity,
-      measure,
+      otherMeasures,
     } = this.ingredient;
-
+    var ingredient_measures_attributes; /* eslint-disable-line camelcase */
+    if(otherMeasures) {
+      ingredient_measures_attributes = otherMeasures.data.map(function(unit) { /* eslint-disable-line camelcase */
+        return Object.assign({}, { id: unit.id }, unit.attributes);
+      });
+    } else {
+      ingredient_measures_attributes = [{ name: undefined, quantity: undefined, id: undefined }]; /* eslint-disable-line camelcase */
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This line has a length of 133. Maximum allowed is 120  vue/max-len